ui/pages/providers/add.js

```diff
@@ -1,15 +1,81 @@
-import { useEffect, useState } from 'react'
+import { Fragment, useEffect, useState } from 'react'
+import { Transition, Dialog } from '@headlessui/react'
 import { useRouter } from 'next/router'
+import copy from 'copy-to-clipboard'
 import Head from 'next/head'
 import Link from 'next/link'
 import { useSWRConfig } from 'swr'
-import { InformationCircleIcon, XIcon } from '@heroicons/react/outline'
 import Tippy from '@tippyjs/react'
+import {
+  DuplicateIcon,
+  CheckIcon,
+  InformationCircleIcon,
+  XIcon,
+} from '@heroicons/react/outline'
 
 import { providers } from '../../lib/providers'
 
 import Dashboard from '../../components/layouts/dashboard'
 
+function SCIMKeyDialog(props) {
```

Comment:
instead of passing in props, can we actually list what we are passing in ? It is kind of confusing just leave it as `props`

Reply:
fixed in the component